Backend/purrfect-match-backend/src/main/java/com/purrfectmatch/restservice/Dogs/Dog.java

```diff
@@ -0,0 +1,55 @@
+package com.purrfectmatch.restservice;
+
+import lombok.Data;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDate;
+
+@Data
+@Entity
+class Dog {
+
+   private @Id @GeneratedValue(strategy = GenerationType.IDENTITY) Long id;
+   private String imageName, name, bio, shelterId;
+   private LocalDate bday, dateArrived, euthanizeDate;
+   private int breed;
+   private Gender gender;
+   private FurLength furLength;
+   private boolean spayedNeutured;
+   private Size size;
```

Comment:
Missing @table and @column annotations, do you want to include them? 
shelterId is a string.